server/helpers/errors.js

```diff
@@ -0,0 +1,9 @@
+class UnreachableCaseError extends Error {
```

Comment:
Let's not use this name as it is specific to a TS feature that gives us an assurance that it won't actually be reachable.